pytorch_lightning/core/hooks.py

```diff
@@ -805,6 +809,18 @@ def on_after_batch_transfer(self, batch, dataloader_idx):
         """
         return batch
 
+    @property
+    def prepare_data_per_node(self) -> bool:
```

Comment:
missing docs update ?

Reply:
Thank you. I will update in the next iteration.